app/model/editions/templates/AustralianEdition.scala

```diff
@@ -56,9 +56,9 @@ object AustralianEdition extends EditionDefinitionWithTemplate {
 
   def FrontTopStoriesAu = front(
     "Top stories",
```

Comment:
```suggestion
    "Top Stories",
```

Reply:
Style is lower case on stories. But thanks!